packages/uswds-core/src/styles/functions/utilities/color.scss

```diff
@@ -12,8 +12,7 @@ Derive a color from a color shortcode
 @use "sass:string";
 @use "../../settings" as *;
 @use "../../functions/general";
-@use "../../tokens/color/shortcodes-color-system" as *;
-@use "../../tokens/color/shortcodes-color-project" as *;
+@use "../../tokens/color/shortcodes-color-all" as *;
```

Comment:
This might require an additional testing. It was previously:

```scss
@use "../../tokens/color/shortcodes-color-system" as *;
@use "../../tokens/color/shortcodes-color-project" as *;
```
The problem was including the new HCM shortcodes as part of system color shortcodes. Happy to revisit if including **all** shortcodes is a problem.